NewApp/ngsa-csharp/Ngsa.DataService/Controllers/GenresController.cs

```diff
@@ -36,8 +41,34 @@ public GenresController(ILogger<GenresController> logger)
         [HttpGet]
         public async Task<IActionResult> GetGenresAsync()
         {
+            // TODO - ILogger
+            Microsoft.AspNetCore.Http.HttpContext context = HttpContext;
+            string message = "Test Error";
+            string key1 = "value1";
+            string key2 = "value2";
+
+            logger.LogError(
+                new EventId(123, nameof(GetGenresAsync)),
+                "{message} {context} {key1} {key2}",
+                message,
+                context,
+                key1,
+                key2);
+
+            // TODO - custom logger
+            Logger.LogError(
+                new EventId(123, nameof(GetGenresAsync)),
+                "Test Error",
```

Comment:
message is the same as string literal.  Maybe just use the variable?

Reply:
resolved